app/src/main/res/values/strings.xml

```diff
@@ -916,62 +916,62 @@
-    <string name="firewall_rule_bypass_app_rules_ip_desc"><![CDATA[This connection was allowed because the <i>destination IP</i> is set to bypass all app-specific rules.<br/><br/> To change go to <i>App-specific Firewall</i> screen.]]></string>
-    <string name="firewall_rule_bypass_universal_ip_desc"><![CDATA[This connection was allowed because the <i>destination IP</i> is set to bypass all universal firewall rules.<br/><br/> To change go to <i>IP Rules<i> in <i>Universal</i> tab.]]></string>
+    <string name="firewall_rule_block_app_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i>.<br/><br/> Go to the <i>All Apps</i> tab to change this.]]></string>
+    <string name="firewall_rule_new_app_desc"><![CDATA[This app made connection for the <i>first</i> time.<br/><br/> Go to <i>Universal</i> tab to change this.]]></string>
+    <string name="firewall_rule_block_app_unmetered_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i> when trying to connect on <i>unmetered</i> network.<br/><br/> Go to the <i>All Apps</i> tab to change this.]]></string>
+    <string name="firewall_rule_block_app_mobile_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i> when trying to connect on <i>metered</i> network.<br/><br/> Go to the <i>All Apps</i> tab to change this.]]></string>
+    <string name="firewall_rule_block_app_univ_mobile_desc"><![CDATA[A user-set universal firewall rule blocked this <i>app</i> when trying to connect on <i>Metered</i> (Mobile) network.<br/><br/> Go to the <i>Universal</i> tab to change this.]]></string>
+    <string name="firewall_rule_lockdown_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i> connecting to other than <i>bypassed IPs</i>.<br/><br/> Go to the <i>All Apps</i> tab to change this.]]></string>
+    <string name="firewall_rule_block_ip_univ_desc"><![CDATA[A user-set firewall rule blocked the <i>destination IP / Port</i>.<br/><br/> Go to the <i>Universal</i> tab to change this.]]></string>
+    <string name="firewall_rule_block_ip_desc"><![CDATA[A user-set firewall rule blocked the <i>destination IP</i>.<br/><br/> Go to the <i>App-specific firewall</i> screen to change this.]]></string>
+    <string name="firewall_rule_bypass_app_rules_ip_desc"><![CDATA[This connection was allowed because the <i>destination IP</i> is set to bypass all app-specific rules.<br/><br/> Go to <i>App-specific Firewall</i> screen to change this.]]></string>
+    <string name="firewall_rule_bypass_universal_ip_desc"><![CDATA[This connection was allowed because the <i>destination IP</i> is set to bypass all universal firewall rules.<br/><br/> Go to <i>IP rules<i> in the <i>Universal</i> tab to change this.]]></string>
     <string name="firewall_rule_device_lock_desc"><![CDATA[This connection was firewalled because <i>device was locked</i> at the time.<br/><br/> To change this behaviour go to <i>Universal</i> tab.]]></string>
```

Comment:
```suggestion
    <string name="firewall_rule_device_lock_desc"><![CDATA[This connection was firewalled because <i>device was locked</i> at the time.<br/><br/> Go to the <i>Universal</i> tab to change this.]]></string>
```